pkg/virt-config/featuregate/inactive.go

```diff
@@ -102,6 +104,7 @@ func init() {
 	RegisterFeatureGate(FeatureGate{Name: VolumeMigration, State: GA})
 	RegisterFeatureGate(FeatureGate{Name: DisableCustomSELinuxPolicy, State: GA})
 	RegisterFeatureGate(FeatureGate{Name: AutoResourceLimitsGate, State: GA})
+	RegisterFeatureGate(FeatureGate{Name: ClusterProfiler, State: GA})
```

Comment:
Being GA I think the feature is active

Reply:
Kinda ticky: the feature is active but disabled